client/src/components/JurisdictionAdmin/BatchRoundDataEntry.tsx

```diff
@@ -0,0 +1,645 @@
+                )}
+              </td>
+            </tr>
+          ))}
+        </tbody>
+      </BatchResultTallySheetTable>
+
+      <Actions>
+        {!isTotalsSheet && (
+          <SheetActions>
+            {isEditing && sheets.length > 1 && !isRenamingSheet && (
+              <>
+                <Button icon="edit" minimal onClick={initiateSheetRename}>
+                  Rename sheet
+                </Button>
+                <Button
+                  icon="delete"
+                  intent="danger"
+                  minimal
+                  onClick={deleteSheet}
+                >
+                  Delete sheet
+                </Button>
+              </>
+            )}
+            <input
+              className={classnames(
+                Classes.INPUT,
+                Classes.SMALL,
+                errors.resultTallySheets?.[index]?.name &&
+                  Classes.INTENT_DANGER,
+                // Visually hide this input instead of completely unmounting it to avoid
+                // interfering with react-hook-form's state management
+                !isRenamingSheet && HIDDEN_INPUT_CLASS_NAME
+              )}
+              defaultValue={sheets[index].name}
+              name={`resultTallySheets[${index}].name`}
+              onChange={() => trigger(`resultTallySheets[${index}].name`)}
+              ref={register({
+                required: true,
+                validate: () => {
+                  const sheetNames = sheets.map((_, i) =>
+                    getValues(`resultTallySheets[${i}].name`)
+                  )
+                  if (new Set(sheetNames).size < sheetNames.length) {
+                    return 'Must be unique'
+                  }
+                  return true
+                },
+              })}
+            />
+            {isRenamingSheet && (
+              <>
+                <ConfirmSheetRenameButton
+                  disabled={Boolean(errors.resultTallySheets?.[index]?.name)}
+                  icon="tick"
+                  minimal
+                  onClick={confirmSheetRename}
+                >
+                  Done
+                </ConfirmSheetRenameButton>
+                <Button icon="delete" minimal onClick={cancelSheetRename}>
+                  Cancel
+                </Button>
+                <ErrorMessage
+                  errors={errors}
+                  name={`resultTallySheets[${index}].name`}
+                  render={({ message }) => (
+                    <SheetRenameErrorMessage>{message}</SheetRenameErrorMessage>
+                  )}
+                />
+              </>
+            )}
+          </SheetActions>
+        )}
+
+        <Spacer />
+
+        {isEditing || formState.isSubmitting ? (
+          <Button
+            disabled={isRenamingSheet}
+            icon="tick"
+            intent="primary"
+            loading={formState.isSubmitting}
+            onClick={saveResults}
+          >
+            Save Results
```

Comment:
We're standardizing on Tally/Tallies instead of Result/Results for user-facing copy. There are a few places that should be changed